guides/developer/plugin-creators/deck-plugin.md

```diff
@@ -0,0 +1,451 @@
+---
+layout: single
+title: "Test a Pipeline Stage Plugin"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.20.6" %}
+
+{% include toc %}
+
+This guide explains how to set up a local Spinnaker environment on your Mac or Windows environment so you can test the `pf4jStagePlugin`, which has both Orca and Deck components. Spinnaker services running locally communicate with the other Spinnaker services running in a local VM. Although this guide is specific to the `pf4jStagePlugin`, you can adapt its contents to test your own plugin.
+
+Example Spinnaker setup:
+
+* OSX or Windows using IP 192.168.64.1 and the VM using 192.168.64.5
```

Comment:
It looks a little bit like you need these IPs to complete the tutorial. You could template them as `{my-ip}` and `{my-vm-ip}` and use those through the tutorial instead of specific addresses.